Src/Fido2.Models/Fido2Configuration.cs

```diff
@@ -56,20 +56,17 @@ public Fido2Configuration()
     /// <summary>
     /// Server origins, including protocol host and port.
     /// </summary>
-    public ISet<string> Origins
+    public IReadOnlySet<string> Origins
     {
         get
         {
             if (_origins == null)
             {
-                _origins = new HashSet<string>();
-
-                // Since we're depricating Origin we ease the transition to move the value automatically, unless its null
+                // Since we're deprecating Origin we ease the transition to move the value automatically, unless its null
 #pragma warning disable CS0618 // Type or member is obsolete
-                if (Origin != null)
-                {
-                    _origins.Add(Origin);
-                }
+                _origins = Origins != null
```

Comment:
Looks good, but small misstake null checking `Origins` instead of `Origin`

Reply:
Guess I shouldn't work on official projects this late. Thanks for keeping an eye out.